### ISSUE TITLE
Peer learning activities

### ISSUE THREAD

**Issue by SweetPotato0213** [nus-cs2030/2021-s2#341]
How to complete this part of assessment: Class participation and peer learning activities (5%)?

**Comment by seanlowjk** [comment]
Hi @SweetPotato0213 

Partially it will be given by your tutors and from the forum and wiki participation and mentioned in the LumiNUS page.

**Comment by charmainechng** [comment]
@seanlowjk Hello,, just to check, is participation/ submission of recitation code under this part of the assessment as well?

**Comment by KevinTan1203** [comment]
@charmainechng Yes. From what I've been told, submitting recitation attempts (be it whether the code is right or wrong) counts as part of your overall class participation.

**Comment by High325** [comment]
Part of the 5% is answering and asking questions on this forum so no worries, it looks like you're well on your way to getting it!

**Comment by SweetPotato0213** [comment]
> Part of the 5% is answering and asking questions on this forum so no worries, it looks like you're well on your way to getting it!

Should we declare our posts in GitHub to tutors or prof?

**Comment by DanielSjtea** [comment]
@SweetPotato0213 I don't believe we need to, as long as you have given your Github username to your TAs it should be fine.
